src/main/java/net/mcreator/element/types/Block.java

```diff
@@ -94,6 +94,9 @@
 	public boolean tickRandomly;
 
 	public boolean isReplaceable;
+	public boolean emitsRedstone;

```

Comment:
AFAIK this parameter controls if redstone connects to the block or not, so consider reworking the UI to indicate this, as the block can provide power even if redstone is not connected, indirectly

Reply:
Done. I specified it in the help boxes.

Reply:
@Max094Reikeb  refactor the field names too

Reply:
What do you mean?

Reply:
emitsRedstone to canProvidePower for UI fields and backend fields

Reply:
canProvidePower already exists...

Reply:
You are correct. But this key actually controls can connect, not canProvidePower. So what I suggest is that you rename the current field canProvidePower to canRedstoneConnect, and rename emitsRedstone to canProvidePower.

Converter is not needed as this PR is more or less redstone functionality redesign.